web/portal/brand/src/entities/Carrier/Carrier.tsx

```diff
@@ -109,14 +109,25 @@ const Carrier: EntityInterface = {
   path: '/carriers',
   toStr: (row: CarrierPropertyList<EntityValues>) => `${row.name}`,
   properties,
-  columns: [
-    'name',
-    'description',
-    'transformationRuleSet',
-    'balance',
-    'proxyTrunk',
-    'statusIcon',
-  ],
+  columns: (state) => {
+    const { features } = state.clientSession.aboutMe.profile || {};
+
+    const billingExists = features.includes('billing');
+
+    const columns = [
+      'name',
+      'description',
+      'transformationRuleSet',
+      'proxyTrunk',
+      'statusIcon',
+    ];
+
+    if (billingExists) {
+      columns.push('balance');
```

Comment:
Column position must not change. Balance should be the 4th column, not the last one